tfhub_dev/assets/rishit-dagli/chest-xray-covid/1.md

```diff
@@ -0,0 +1,8 @@
+# Placeholder rishit-dagli/chest-xray-covid/1
+Lightweight deep learning model for clasifying chest xrays as COVID or normal.
```

Comment:
I assume this is a binary classifier, could we rephrase this as `COVID or not` instead of `COVID or normal`. Same for the rest of the MD files.

Reply:
Sure! That seems to be a better way of writing this.